### ISSUE TITLE
Lightgun (GunCon 3) doesn't work with SNAC controller enabled

### ISSUE THREAD

**Issue by phantomlord2oo** [MiSTer-devel/NES_MiSTer#332]
Having an issue where if I have the GunCon3 plugged in (via USB) you can only set it to Player 2 when SNAC is disabled. I tried other cores (Genesis, SNES) and I can have a SNAC controller enabled while using the lightgun. 

Does enabling SNAC on this core automatically occupy both ports (player 1 and player 2)? That may be causing the issue.

**Comment by Kitrinx** [comment]
Yes, SNAC is a controller port and the light gun's wire is always connected to slot 2, because all light gun games use port 2. This is expected behavior.

**Comment by phantomlord2oo** [comment]
But why can't I have SNAC enabled and use the GunCon3 (assigned to JoyPort 2) at the same time? I can only use the GunCon3 if I disable SNAC. No other core has this issue.

**Comment by Kitrinx** [comment]
because they both compete for the same (single) wire. SNAC uses that wire, so does virtual lightgun support. There is only one wire for light guns.

**Comment by phantomlord2oo** [comment]
Ahhh I gotcha. I see in the OSD is says Controllers (plural). Guess I'll just disable SNAC when I want to the zapper. Thanks for clearing this up for me!

**Comment by Kitrinx** [comment]
on nes, for compatibility with the famicom, SNAC's wires used for the gun ALWAYS go to port 2. The famicom had hard wired controllers with only an expansion port, and that's how it was wired.
